src/modules/subcommands/handleBreakout.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable jsdoc/require-param */
-import { GuildChannel, VoiceChannel } from "discord.js";
+import { VoiceChannel } from "discord.js";
```

Comment:
```suggestion
import { GuildMember, VoiceChannel } from "discord.js";
```
Only if the suggestion on line 24 is accepted.

Reply:
I could not see that suggestion? Maybe I am just tired 🤦‍♂️

Reply:
You committed it 🙂

Reply:
ok 😂